### PR TITLE
interactive flow maps vignette

### PR DESCRIPTION
**Pull request by e-kotov**

### CODE REVIEW

**Comment by e-kotov** [comment]
will resolve manually

File: vignettes/flowmaps-interactive.qmd [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+
+
+```{r}
+library(spanishoddata)
+library(flowmapblue)
+library(tidyverse)
+library(sf)
+```
+
+{{< include ../inst/vignette-include/setup-data-directory.qmd >}}
```
Comment (Robinlovelace) [inline vignettes/flowmaps-interactive.qmd:41]: So handy!

**Review by Robinlovelace — APPROVED** [review]
Great new vignette!

File: man/figures/barcelona-time.gif
[the remark this answers is not in the document]
Comment (e-kotov) [reply man/figures/barcelona-time.gif:1]: @Robinlovelace  I exclude all animations with Rbuildignore and replace with static images. So not an issue at all. Animations only go to github and into pkgdown

File: vignettes/flowmaps-interactive.qmd (hunk 1 above)
Comment (e-kotov) [reply vignettes/flowmaps-interactive.qmd:41]: indeed! it took a while to figure out where to put the qmd to make both pkgdown and r cmd check happy

File: man/figures/barcelona-time.gif
[the remark this answers is not in the document]
Comment (e-kotov) [reply man/figures/barcelona-time.gif:1]: Ahh, there may be a problem with CRAN complaining about missing images, as these will be linked in the MD file. But we can replace them with direct links to repo or to the pkgdownwebsite

**Pull request merged**